timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -219,26 +220,20 @@ void declare_context(py::module &m) {
                 if (burn_in < 0) {
                     throw std::runtime_error("burn in steps must be greater than zero");
                 }
+                if (radius < 0.0) {
+                    throw std::runtime_error("radius must be greater or equal to zero");
+                }
+                if (k <= 0.0) {
+                    throw std::runtime_error("k must be greater than zero");
+                }
+
```

Comment:
behavior: is there a maximum value of `k` to enforce as well? (limit may be imposed by MD stability)

Reply:
I would guess that it is imposed by MD stability, but haven't come across a value I would codify. Though we could say 100k, no strong reason to think we would want to go that high.

Reply:
Maybe 10x or 100x the max value of `k` you've tested, with a reminder to determine a threshold more precisely in the future?

(Confident we want to rule out `k == +inf`, but haven't attempted to determine a finite threshold.)